### PR TITLE
Upgrade hero background and add Github link to navigation

### PR DESCRIPTION
**Pull request by jasonramirez**
screen shot 2015-11-04 at 5 17 00 pm

### CODE REVIEW

**Comment by seanpdoyle** [comment]
:heart_eyes:

File: _sass/_hero.scss [hunk 1]
```diff
@@ -1,26 +1,35 @@
 .hero {
   @include size(100vw 100vh);
   align-items: center;
-  background-color: $ember-orange;
+  background-image: url("/images/hero-background-triangles.svg");
```
Comment (joshuaogle) [inline _sass/_hero.scss]: Does `image-url()` not work here?

**Comment by joshuaogle** [comment]
I may be missing something, but why would `images/hero-background.jpg` not just simply be a CSS gradient?

Overall this looks great! :+1:

**Comment by jasonramirez** [comment]
@joshuaogle great catch! The `hero-background.jpg` image is not actually being used. I went the way of the gradient, and forgot to delete this.

**Pull request merged**